com.github.hneemann.digital/com.github.hneemann.digital.json

```diff
@@ -0,0 +1,123 @@
+      "org.freedesktop.Sdk.Extension.openjdk21"
+  ],
+  "command": "Digital.sh",
+  "finish-args": [
+      "--persist=.java",
+      "--share=ipc",
+      "--share=network",
+      "--socket=x11",
+      "--socket=wayland",
+      "--filesystem=home"
+  ],
+  "cleanup": [
+      "/include",
+      "/lib/pkgconfig",
+      "/share/aclocal",
+      "/share/man",
+      "*.la", "*.a"
+  ],
+  "build-options": {
+      "no-debuginfo": true
+  },
+  "modules": [
+        {
+            "name" : "openjdk",
+            "buildsystem" : "simple",
+            "build-commands" : [ "/usr/lib/sdk/openjdk21/install.sh" ]
+        },      
+        {
+            "name": "digital",
+            "buildsystem": "simple",
+            "build-commands": [
+                "echo Building Digital",
+                "mkdir -p /app/bin",
+                "mkdir -p /app/share/Digital",
+                "cp -r docu /app/share/Digital",
+                "cp -r examples /app/share/Digital",
+                "cp -r lib /app/bin",
+                "install -Dm755 Digital.jar /app/bin/Digital.jar",
+                "install -Dm755 Digital.sh /app/bin/Digital.sh"               
+            ],
+            "sources" : [
+                {
+                    "type": "archive",
+                    "url": "https://github.com/hneemann/Digital/releases/download/v0.31/Digital.zip",
+                    "sha512": "e3998e92e47d71caaf34b05be51bb623353806ea58c5018046665903771f88839bb5c2c6ada4ace11ec99ffa091328b414941e070aee343a590a3e566620af50"
+                },
+                {
+                    "type": "file",
+                    "path": "Digital.sh"
+                }
+            ]
+        },
+        {
+            "name": "iverilog",
+            "buildsystem": "autotools",
+            "sources": [
+                {
+                    "type": "script",
+                    "commands": [
+                        "#!/bin/sh",
+                        "autoconf"
+                    ]
+                },
+                {
+                    "type": "archive",
+                    "url": "https://github.com/steveicarus/iverilog/archive/refs/tags/v12_0.tar.gz",
+                    "sha512": "a8e112f760903335ef2fbf00529e5fe282b83c1f583c4316373b9410d7fad2a53d5c4adb82dd62491c14a620cb5b1fab117cb0cde638c86b65fe5b7b0dd07152"
+                }
+            ]
+        },
+		{
+			"name": "ghdl",
+			"buildsystem": "simple",
+			"build-commands": [
+				"echo installing ghdl",
+				"find bin -type f -exec install -D {} /app/{} \\;",
+				"find lib -type f -exec install -D {} /app/{} \\;"
+			],
+			"sources": [
+				{
+					"type": "archive",
+					"url": "https://github.com/ghdl/ghdl/releases/download/v4.1.0/ghdl-gha-ubuntu-22.04-mcode.tgz",
+					"sha512": "bb0f58d3585e84f9b0ab18846772a4bf7693bc0186998982428ae251d623d28544acf62ef0a9d8237e96cf07f91432fb510b356c33c7981df546e320c9b1612f"
+				}
+			]
+		},
+        {
+            "name": "digital-desktop-integration",
```

Comment:
merge the sources here with the module for your app and drop this. Put the install commands in post install.